.changeset/silent-ducks-act.md

```diff
@@ -0,0 +1,7 @@
+---
+"@frontify/guideline-blocks-settings": patch
```

Comment:
```suggestion
"@frontify/guideline-blocks-settings": minor
```

Reply:
There are no breaking changes, is there? 🤔

Reply:
also its not just a "bug fix", imo minor fits better